components/core/src/error.rs

```diff
@@ -340,6 +347,7 @@ impl fmt::Display for Error {
             Error::GetExitCodeProcessFailed(ref e) => e.to_string(),
             Error::CreateToolhelp32SnapshotFailed(ref e) => e.to_string(),
             Error::WaitForSingleObjectFailed(ref e) => e.to_string(),
+            Error::TabWriterIntoInnerFailed(ref e) => e.to_string(),
```

Comment:
See other comments. I dont think this type should be needed.

Reply:
Removed in https://github.com/habitat-sh/habitat/pull/7841/commits/78f9d5e6a2b76e641bb6f2a2971c62744b5e762b